src/main/java/seedu/address/model/person/ModuleTagSet.java

```diff
@@ -0,0 +1,146 @@
+package seedu.address.model.person;
```

Comment:
This has 9/10 method coverage and I have no idea which is the violating method. Stress.